src/Shared/helperFunctions.ts

```diff
@@ -0,0 +1,25 @@
+import { States } from './state';
+import MFBLogoCO from '../Assets/Logos/mfb_colorado_logo.png';
+import MFBLogoNC from '../Assets/Logos/mfb_nc_logo.png';
+import MFBDefaultLogo from '../Assets/Logos/mfb_default_logo_header.png';
+import { coLogoAlt, coLogoSource, coShareLink, coTwoOneOneLink, ncLogoAlt, ncLogoSource } from '../Components/Referrer/referrerDataInfo';
+
+export const getLogo = (state: States): string => {
```

Comment:
We went this route, because when we added the path of the logo to the config and retrieve it in the footer (anywhere), the image was broken and it only was working when we imported the file manually
![image](https://github.com/user-attachments/assets/467e84e6-d2ae-4002-92e8-1009101a4a5e)

Reply:
Can you do it like we do the icons in https://github.com/Gary-Community-Ventures/benefits-calculator/blob/main/src/Components/Config/configHook.tsx